features/adb-openshift.feature

```diff
@@ -57,6 +57,14 @@ Feature: Command output from various OpenShift related commands in ADB
     Then the exit status should be 0
     And the binary "oc" should be installed
 
+    When I run `bundle exec vagrant service-manager install-cli openshift --cli-version 1.3.0`
+    Then the exit status should be 0
+    And the binary "oc" of service "openshift" should be installed with version "1.3.0"
+
+    When I evaluate and run `bundle exec vagrant service-manager install-cli openshift --path #{ENV['VAGRANT_HOME']}/oc`
```

Comment:
👍 I like how the tests look like now. Also nice to use the same path in the _When_ and _Then_ clause and that it gets transparently interpolated.